src/validations/rules/sap.sch

```diff
@@ -586,6 +586,43 @@
                 FedRAMP SAP has a metadata/party with an @name of 'iso-iec-17020-identifier' has a correctly formatted @value.</sch:assert>
         </sch:rule>
     </sch:pattern>
+    
+    <sch:pattern id="assessment-assets">
+        <sch:rule
+            context="oscal:assessment-assets/oscal:component[@type = 'software']">
+            <sch:assert
+                diagnostics="has-software-component-vendor-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.13"
+                fedramp:specific="true"
+                id="has-software-component-vendor"
+                role="error"
+                test="oscal:prop[@name = 'vendor']">This FedRAMP SAP assessment asset component has a vendor name.</sch:assert>
+
+            <sch:assert
+                diagnostics="has-software-component-tool-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.13"
+                fedramp:specific="true"
+                id="has-software-component-tool"
+                role="error"
+                test="oscal:prop[@name = 'name']">This FedRAMP SAP assessment asset component has a tool name.</sch:assert>
+
+            <sch:assert
+                diagnostics="has-software-component-version-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.13"
+                fedramp:specific="true"
+                id="has-software-component-version"
+                role="error"
+                test="oscal:prop[@name = 'version']">This FedRAMP SAP assessment asset component has a version number for the tool.</sch:assert>
+
+            <sch:assert
+                diagnostics="has-software-component-status-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.13"
+                fedramp:specific="true"
+                id="has-software-component-status"
+                role="warning"
+                test="oscal:status[@state='operational']">This FedRAMP SAP assessment asset component has a status with a state of 'operational'.</sch:assert>
```

Comment:
The Guide note about this states: "OSCAL syntax requires a status field within each component assembly. For assessment tools, the state should typically be 'operational'"

I took that to mean that it could be another value.  I expect that another value would necessitate a remark element as well, but that isn't stated in the guide.

Reply:
Huh, that's an odd one. A remark would seem to make sense.